test/inductor/test_cutlass_evt.py

```diff
@@ -0,0 +1,216 @@
+# Owner(s): ["module: inductor"]
+import unittest
+
+from torch._dynamo.test_case import TestCase
+from torch._inductor.codegen.cuda.cutlass_utils import try_import_cutlass
+from torch.testing._internal.inductor_utils import HAS_CPU, HAS_CUDA
+
+
+if try_import_cutlass():
+    import cutlass_library as cutlass_lib
+    from cutlass_library import EpilogueScheduleType
+
+    LayoutType = cutlass_lib.LayoutType
+    DataType = cutlass_lib.DataType
+    from torch._inductor.codegen.cuda.cutlass_lib_extensions.evt_extensions import (
+        CutlassTensor,
+        trace,
+    )
+
+    class MockTileDescription:
+        threadblock_shape = (128, 128, 8)
+
+
+class TestCutlassEVT(TestCase):
+    @unittest.skipIf(not try_import_cutlass(), "requires cutlass")
+    def test_evt_codegen(self):
```

Comment:
Have you verified this is running in fbcode ci ?

Reply:
I will try after I land this actually, I don't want ghimport messing w/ the stack lol